### ISSUE TITLE
Is `sbi_cppc_read_hi` allowed to ignore reserved/unimplemented register IDs?

### ISSUE THREAD

**Issue by repnop** [riscv-non-isa/riscv-sbi-doc#132]
The documentation for `sbi_cppc_read_hi` currently reads:

> This function always returns zero in `sbiret.value` when supervisor mode XLEN is 64 or higher.

Does this imply no validity checking is done when called with `XLEN` >= 64, or should implementations still return an error on a reserved or unimplemented register ID?

**Comment by atishp04** [comment]
Yes. No validity checking is required when sbi_cppc_read_hi is called with XLEN >= 64. The additional validity checking is unnecessary because the sbi_cppc_read_hi can't provide any meaningful value when XLEN>=64 . There is no point in burdening the implementation with additional sanity checks. The invalid register would have already checked with cppc_probe call also.
